internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,431 @@
+		}
+		q := &survey.Question{
+			Name: "dbUsername",
+			Prompt: &survey.Input{
+				Message: message,
+				Help:    usage.DBUsername,
+				Default: usrDefault,
+			},
+			Validate: func(val interface{}) error {
+				username, _ := val.(string)
+				user, err := opts.store.DatabaseUser(convert.AdminDB, opts.ConfigProjectID(), username)
+				if err != nil {
+					if !strings.Contains(err.Error(), fmt.Sprintf("No user with username %s exists.", username)) {
+						return err
+					}
+				}
+
+				if user != nil {
+					return errors.New("a user with this username already exists")
+				}
+
+				return nil
+			},
+		}
+
+		qs = append(qs, q)
+	}
+
+	if opts.DBUserPassword == "" {
+		q := &survey.Question{
+			Name: "dbUserPassword",
+			Prompt: &survey.Password{
+				Message: "Insert the Password for authenticating to MongoDB [Press Enter to use an auto-generated password]",
+				Help:    usage.Password,
+			},
+		}
+
+		qs = append(qs, q)
+	}
+
+	return qs
+}
+
+func (opts *Opts) newAccessListQuestion() *survey.Question {
+	if opts.IPAddress != "" {
+		return nil
+	}
+
+	message := "Insert the IP entry to add to the Access List"
+	//nolint:ifshort //false positive
+	publicIP := net.IPAddress()
+
+	if publicIP != "" {
+		message = fmt.Sprintf("Insert the IP entry to add to the Access List [Press Enter to use your public IP address '%s']", publicIP)
+	}
+
+	return &survey.Question{
+		Name: "ipAddress",
+		Prompt: &survey.Input{
+			Message: message,
+			Help:    usage.AccessListIPEntry,
+			Default: publicIP,
+		},
+	}
+}
+
+func (opts *Opts) newClusterQuestions() []*survey.Question {
+	var qs []*survey.Question
+
+	if opts.ClusterName == "" {
+		clusterName := opts.newClusterName()
+		message := "Insert the cluster name"
+		if clusterName != "" {
+			message = fmt.Sprintf("Insert the cluster name [Press Enter to use the auto-generated name '%s']", clusterName)
+		}
+		q := &survey.Question{
+			Name: "clusterName",
+			Prompt: &survey.Input{
+				Message: message,
+				Help:    usage.ClusterName,
+				Default: clusterName,
+			},
+		}
+		qs = append(qs, q)
+	}
+
+	if opts.Provider == "" {
+		q := &survey.Question{
+			Name: "provider",
+			Prompt: &survey.Select{
+				Message: "Insert the cloud service provider on which Atlas provisions the hosts",
+				Help:    usage.Provider,
+				Options: []string{"AWS", "GCP", "AZURE"},
+			},
+		}
+		qs = append(qs, q)
+	}
+
+	return qs
+}
+
+func (opts *Opts) newRegionQuestions() *survey.Question {
+	if opts.Region != "" {
+		return nil
+	}
+	return &survey.Question{
+		Name: "region",
+		Prompt: &survey.Select{
+			Message: "Insert the physical location of your MongoDB cluster",
+			Help:    usage.Region,
+			Options: DefaultRegions[strings.ToUpper(opts.Provider)],
+		},
+	}
+}
+
+// dbUsername returns the username of the user by running the command 'whoami'
+func dbUsername() string {
+	command := "whoami"
+	cmd := exec.Command("bash", "-c", command)
```

Comment:
bash won't work on windows (and I just tested this)

Reply:
mh... This is strange. I tested on windows (using msi) and it worked: It is the Administrator user that you see here 

![Screenshot 2021-03-05 at 14 24 59](https://user-images.githubusercontent.com/5663078/110128341-957a2380-7dbe-11eb-9f49-385bb3621f96.png)

Reply:
depends if you have something like `mingw`, which I don't, and we should cater for that kind of users as well, I went quite basic even using `cmd` over `powershell` and I have no dev tools installed

Reply:
+1 to guss's point - you'll need to call a windows API to get the username. 
tbqh, the unix version of this should also be updated to call a unix API to get the username rather than running a shell :) 

this looks promising: https://golangcode.com/get-the-current-username-name-and-home-dir-cross-platform/ 

that said: you could merge this like this & do that change in a separate ticket